pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -87,6 +88,19 @@ ceph -n client.admin auth get-or-create osd."$OSD_ID" mon 'allow profile osd' mg
 if [[ "$CV_MODE" == "lvm" ]]; then
 	TMP_DIR=$(mktemp -d)
 
+	# "ceph.conf" must have the "fsid" global configuration after the following Ceph's PR is merged.
+	# https://github.com/ceph/ceph/commit/25655e5a8829e001adf467511a6bde8142b0a575
+	cp -a /etc/ceph /tmp/
```

Comment:
This is only needed if we enter the first condition right? Why not just do this all the time and create a ceph.conf? Can we use `TMP_DIR` instead of `/tmp`?

Reply:
> This is only needed if we enter the first condition right? Why not just do this all the time and create a ceph.conf?

Indeed.

> Can we use TMP_DIR instead of /tmp?

Yes, I just overlooked TMP_DIR. However, the new version of this PR won't use a temporary directory. Just create a ceph.conf from rook-config-override cm and add the fsid field if not exist.